lib/PuppeteerSharp/Frame.cs

```diff
@@ -97,6 +97,28 @@ internal Task<ElementHandle> AddScriptTag(dynamic options)
             throw new NotImplementedException();
         }
 
+        internal async Task<string> GetContentAsync()
+        {
+            var result = await EvaluateAsync(@"() => {
```

Comment:
I know this is a string, but I would try to indent it to follow the code, it looks off at first sight.